src/app/page.tsx

```diff
@@ -5,13 +5,15 @@ import {
 	HowItWorksSection,
 	RoadmapSection,
 } from "@/components/04-templates";
+import { CardStats } from "@/components/02-molecules";
 
 export default function Home() {
 	return (
 		<main className="flex h-full w-full flex-col items-center justify-center">
 			<HeroSection />
 			<PartnersSection />
 			<WhyChooseUsSection />
+			<CardStats />
```

Comment:
hey yo @Hardsys-Andre !  Should move the CardStats to ``ChooseUs`` organism
because it's an atom and should be part of the ``WhyChooseUsSection`` ( Atomic Design )